projects/packages/jetpack-mu-wpcom/src/features/launch-button/index.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Adds a "launch site" button to the admin bar.
+ */
+
+use Automattic\Jetpack\Status;
+
+/**
+ * Adds a "launch site" button to the admin bar.
+ * 
+ * @param WP_Admin_Bar $admin_bar The WordPress admin bar.
+ */
+function wpcom_add_launch_button_to_admin_bar( WP_Admin_Bar $admin_bar ) {
+	$is_launched = true;
+	$blog_id = get_current_blog_id();
+
+	if ( defined( 'IS_ATOMIC' ) && IS_ATOMIC ) {
```

Comment:
Since you use the `Status` package in that file already, you could switch to using `is_woa_site` from `Status\Host`